CHANGES

```diff
@@ -9,14 +9,19 @@
 
  Changes between 1.1.0k and 1.1.0l [xx XXX xxxx]
 
+  *) Compute ECC cofactors if not provided during EC_GROUP construction. Before
+     this change, EC_GROUP_set_generator would accept order and/or cofactor as
+     NULL. After this change, only the cofactor parameter can be NULL. It also
+     does some minimal sanity checks on the passed order.
```

Comment:
We should include the CVE number in the CHANGES entry (like the entry below). I guess we missed this from the 1.1.1. version.

Reply:
Fixed by dbbdea6d45b177167fc440d9a96d85579f015ed3